pkg/apis/contrail/v1alpha1/templates/templates_helper_functions.go

```diff
@@ -0,0 +1,49 @@
+package templates
```

Comment:
Cannot these separate functions be more universal?
For example I've created something like that:
```
func joinServerList(servers []string, port *int, sep string) string {
	if port != nil {
		strPort := strconv.Itoa(*port)
		return strings.Join(servers, ":"+strPort+sep) + ":" + strPort
	}
	return strings.Join(servers, sep)

```

Because in case that we'd need to use different separator or sth then we'll have to make new function

Reply:
Done. Left the separate functions, but they both call a common function that accepts the separator as the argument.